tests/test_retry_mechanism.py

```
@@ -0,0 +1,281 @@
+            Exception("Rate limit exceeded"),  # 初回エラー
+            [Mock(content="テスト応答")]        # リトライ後成功
+        ]
+        
+        # タスクインスタンスを作成
+        task_instance = stream_chat_response
+        task_instance.request = Mock()
+        task_instance.request.id = 'test-task-id'
+        task_instance.request.retries = 0
+        
+        # 1回目の実行（エラー）
+        with self.assertRaises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)